app/setup/styleguide.styles.js

```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
```

Comment:
- [ ] I would keep this to the left. Else you could miss the titles on wide screens.